src/index.html

```diff
@@ -4,9 +4,356 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo">
+          <a href="#" class="logo-link">
+            <img src="/src/images/logo.png" alt="header-logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li>
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
+            <li >
```

Comment:
```suggestion
            </li>
            
            <li >
```
Add one empty line between sibling blocks inside the same parent block for both of them.